00-HTML-CSS-basics/Ex02/index.html

```diff
@@ -0,0 +1,69 @@
+            </header>
+            <article>
+                <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut pretium velit velit, vitae sodales lectus maximus id. Duis et lectus nulla. Proin maximus leo quam, in porttitor risus blandit et. Nunc vehicula justo sit amet magna porttitor vestibulum. Vivamus vel nisi tempus, viverra turpis et, dapibus eros. Sed et tortor tellus. Vivamus vitae metus non purus sagittis aliquam id quis purus. Vestibulum lacus risus, pellentesque a aliquet ut, euismod et libero. Donec lobortis orci fermentum, aliquet tellus ac, convallis velit.
+
+Nulla facilisi. Nullam et viverra lectus. Aenean ultricies eros in lorem tincidunt, aliquam condimentum orci venenatis. Etiam fermentum, leo eget fringilla aliquam, lorem metus laoreet lacus, eu bibendum elit nunc vel orci. Maecenas tempus a ipsum ut maximus. In facilisis molestie diam, at laoreet nibh eleifend in. Etiam ullamcorper, leo eget rutrum mollis, augue sapien facilisis velit, nec accumsan nunc nisl vel risus. Pellentesque efficitur orci in augue euismod, quis dictum metus maximus. Nunc vestibulum tortor at enim tempor, consequat iaculis mauris sollicitudin. Maecenas bibendum augue a velit interdum, sit amet imperdiet purus suscipit. Aenean tincidunt nisl nisl, ac dapibus ipsum malesuada eget. Donec ac posuere tortor. Morbi nec sem vel ipsum ultrices molestie a in orci. Cras vel nisi venenatis, interdum lacus eget, hendrerit leo.
+
+Curabitur lobortis hendrerit iaculis. Ut hendrerit elit eget erat feugiat porta. Quisque ut accumsan tellus. Etiam id egestas nisi. Vestibulum sit amet nulla vulputate, congue massa ultricies, fringilla justo. Sed magna enim, posuere vitae ultrices sit amet, ultrices eu enim. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Pellentesque hendrerit purus vitae quam consectetur, bibendum fermentum mauris imperdiet. Aliquam egestas mattis sollicitudin. Quisque at libero ornare, pharetra mi ac, accumsan tellus. Nulla tincidunt et nisl at maximus. Curabitur non sapien quis metus hendrerit ultricies vel eu neque. Interdum et malesuada fames ac ante ipsum primis in faucibus. Integer vulputate ligula lorem, quis eleifend metus sodales in. Nunc leo velit, aliquet non lectus eget, scelerisque sagittis elit. Etiam a congue velit, nec fringilla elit. </p>
+            </article>
+            <footer>
+
+            </footer>
+        </section>
+        <aside class="sidebar">
```

Comment:
I recommend to leave the tags one near the other  when no content is inside. For example, `<aside></aside>`, no need to apply this comment.